README.md

```diff
@@ -17,8 +17,7 @@ attaching them to workloads.
 
 Independent CSI plugins are provided to support RBD and CephFS backed volumes,
 
-- For details about configuration and deployment of RBD and CephFS CSI plugins,
-  see documentation in `docs/`.
+- For details about configuration and deployment of RBD plugin, please refer [this doc](https://github.com/ceph/ceph-csi/blob/master/docs/deploy-rbd.md) and for CephFS plugin please refer [this](https://github.com/ceph/ceph-csi/blob/master/docs/deploy-cephfs.md).
```

Comment:
canm we change `this doc` to `deploy-rbd` and  `this` to `deploy-cephfs` or something else

Reply:
sure.